hypervisor/dm/vpci/vpci.c

```diff
@@ -483,10 +483,24 @@ static int32_t write_cfg(struct acrn_vpci *vpci, union pci_bdf bdf,
 }
 
 /**
+ * @brief Initialize a vdev structure.
+ *
+ * The function vpci_init_vdev is used to initialize a vdev structure with a PCI device configuration(dev_config)
+ * on a specified vPCI bus(vpci). If the function vpci_init_vdev initializes a SRIOV Virtual Function(VF) vdev structure,
+ * the parameter pf_vdev is the VF associated Physical Function(PF) vdev structure, otherwise the parameter pf_vdev is NULL.
+ * The caller of the function vpci_init_vdev should guarantee execution atomically.
+ *
+ * @param vpci       Pointer to a vpci data structure
+ * @param dev_config Pointer to a dev_config structure
+ * @param pf_vdev    Pointer to a SRIOV PF vdev structure for SRIOV VF vdev structure initialization
+ *                   For the normal PCIe vdev structure initialization, parameter pf_vdev is NULL
```

Comment:
How about to rename it as parent_pf or parent_pf_vdev?

Reply:
The comment can be:

If the def_config points to a VF vdev, this parameter indicator the parent PF vdev. Otherwise, it is NULL.

Reply:
OK, I will refine it